test/performance/odp_timer_perf.c

```diff
@@ -45,6 +45,8 @@ typedef struct test_stat_t {
 	uint64_t events;
```

Comment:
The `odp_timer_perf.c` changes should be moved to a separate commit with reasoning why and what is being changed in the respective commit message.

Reply:
Thanks for the comment, would address them in next version.